pkg/agent/flannel/flannel.go

```diff
@@ -71,6 +71,11 @@ func flannel(ctx context.Context, flannelIface *net.Interface, flannelConf, kube
 	go network.SetupAndEnsureIPTables(network.MasqRules(config.Network, bn.Lease()), 60)
 	go network.SetupAndEnsureIPTables(network.ForwardRules(config.Network.String()), 50)
 
+	if flannelIPv6Masq && config.IPv6Network.String() != emptyIPv6Network {
```

Comment:
Could you add a logrus.Debug line with something like `Creating iptable rules for ipv6 node masquerading in flannel`?